### ISSUE TITLE
ENH: Default `steps` in `linspace` 

### ISSUE THREAD

**Issue by wd60622** [pymc-devs/pytensor#923]
### Before

```python
`pt.linspace(0, 10)` # Error
`np.linspace(0, 10)` # Fine
```

### After

```python
Same result as numpy
```

### Context for the issue:

Numpy compat

**Comment by ricardoV94** [comment]
Absolutely, wanna open a PR?

**Comment by jessegrabowski** [comment]
`linspace` is completely refactored in #748 to match numpy's signature and defaults, including the `steps` argument. I'll push to get that over the finish line soon, and it will close this issue.

**Comment by wd60622** [comment]
Is this closed @jessegrabowski ?

**Comment by jessegrabowski** [comment]
Yep, see [here](https://github.com/pymc-devs/pytensor/blob/367351f321f90e1bc38393562e8c9c1e59a9606e/pytensor/tensor/extra_ops.py#L1677). Default is now 50. 

Closed by #748
